### PR TITLE
cross site scripting fixes

### PR DESCRIPTION
**Pull request by ustramooner**
found a few issues when checking sonarqub with BurpSuite

### CODE REVIEW

File: server/sonar-web/src/main/webapp/WEB-INF/app/views/drilldown/issues.html.erb [hunk 1]
```diff
@@ -228,7 +228,7 @@
   window.drilldown = {
     metric: null,
     rule: <% if @rule %>'<%= @rule.key -%>'<% else %>null<% end %>,
-    severity: <% if @severity %>'<%= @severity -%>'<% else %>null<% end %>,
+    severity: <% if @severity %>'<%= @escape_javascript(severity) -%>'<% else %>null<% end %>,
```
Comment (stas-vilchik) [inline server/sonar-web/src/main/webapp/WEB-INF/app/views/drilldown/issues.html.erb]: @ustramooner please fix the typo here, should be `<%= escape_javascript(@severity) -%>`

**Comment by ustramooner** [comment]
@stas-vilchik fixed

**Comment by stas-vilchik** [comment]
@ustramooner still not right, please check my comment again

**Comment by stas-vilchik** [comment]
Thanks for contributing, closed.

**Pull request closed without merging**